### PR TITLE
✨AmpAnalytics: Add linker to ADOBEANALYTICS_CONFIG (#26289)

### LINKED ISSUE #26289: Add linker to ADOBEANALYTICS_CONFIG
## What's the issue?

I'd like to update the Adobe analytics config to take advantage of the linker. I'll also refresh the configuration to use xhrpost and clean up some other issues.

## How do we reproduce the issue?

I'll add something like this to both extensions/amp-analytics/0.1/vendors/adobeanalytics.js and extensions/amp-analytics/0.1/vendors/adobeanalytics.json

'linkers': {
  'enabled': true,
  'proxyOnly': false,
  'linker': {
    'ids': {
      'adobe_amp_id': '${vid}'
    }
  }
}

## What browsers are affected?

All browsers.

## Which AMP version is affected?

This is a new feature.

### PR DESCRIPTION
**Pull request by cory-work**
We'd like to take advantage of the linker in Adobe Analytics. Additionally we made some small updates outlined below:
* Changed the internal Adobe Analytics type from 0 => 2
* removed the request type priority from AdobeAnalytics
* added .python_version to .gitignore
* updated tests for AdobeAnalytics to reflect changes

Fixes #26289

### CODE REVIEW

**Comment by cory-work** [comment]
@googlebot I signed it!

**Comment by cory-work** [comment]
@googlebot I signed it!

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,4 @@ firebase.json
 src/purifier/dist
 build-system/tasks/performance/cache
 build-system/tasks/performance/results.json
+.python-version
```
Comment (zhouyx) [inline .gitignore]: I think you want to keep the change local

Comment (cory-work) [reply .gitignore]: I'm happy to revert this. I found that I needed to downgrade to python 2.7 for something to work (it's been a while so what that was escapes me). I suspect others will encounter this and use pyenv to set a version to do AMP development, so you may want this in the future.

As for the example test, that is a great suggestion. Thanks!

Comment (zhouyx) [reply .gitignore]: good point. cc @rsimha

**Comment by zhouyx** [comment]
The change should not affect the visual diff test, i think it's flaky. cc @Enriqe @gmajoulet to confirm. Thanks.

**Comment by gmajoulet** [comment]
It does look flaky, I'm re-running the visual diff tests.

**Comment by rsimha** [comment]
> It does look flaky, I'm re-running the visual diff tests.

Is there a single visual test that always flakes? If so, we should skip it until it's fixed.

Sending a bat signal to this week's build cop @danielrozenberg (who is also our resident expert on visual tests).

**Comment by gmajoulet** [comment]
I've seen different flakes in the past few hours, mostly linked to images not loading.

**Comment by gmajoulet** [comment]
@zhouyx ready to merge :))

**Comment by zhouyx** [comment]
Woohoo! Thanks all! PR Merged

**Pull request merged**